modules/publication/templates/publication_submission_confirmation.tpl

```diff
@@ -0,0 +1,9 @@
+Subject: [LORIS] Project proposal submission confirmation
+
+Hello,
+This is to confirm your project proposal "{$Title}" received on "{$Date}".
+We will notify you once your proposal has been reviewed. In the mean time, you
+can edit your project proposal through the following link: "{$URL}".
```

Comment:
I don't think the quotation marks are needed around the Date or the URL

Reply:
woops, this actually shouldn't be here at all